sample_apps/simple_app_under_keystone/instructions.md

```diff
@@ -8,12 +8,26 @@ No dependencies on anything, can build on any architecture and with any compiler
 
 ## Example App
 
+Start Qemu
+
+1. `docker pull keystoneenclaveorg/keystone:dev`
+2. `docker run -it --entrypoint /bin/bash keystoneenclaveorg/keystone:dev --volume ABS_CERTIFIER_PATH:/certifier`
+3. `cd /keystone/build && ./scripts/run-qemu.sh`
+   1. Note down the `PORT` Qemu uses
+4. Username: `root` Password: `sifive`
+5. `insmod keystone-driver.ko`
+
 Bare app -- keystone_api only, without certifier:
 
-1. Set up Keystone: todo instructions. Working on updating Docker image.
-2. `cd certifier/sample_apps/simple_app_under_keystone && make -f keystone_example_app.mak bare_app`
+1. `docker ps # find the hash of your Keystone container`
```

Comment:
Do you think it's better to have (1) and (2)? I think we could also assume that these commands are continuous in a single interactive session.

Reply:
I am not aware of a way to do it in a single terminal, since once Qemu starts the commands are stuck with going there. Is there a convenient way to background Qemu and come back to it?

Reply:
I see why this was needed, but we should provide what they're doing with this command, (e.g., open another terminal for the same docker image).
This is a valid way of opening another terminal, but there are other ways of doing that, for example, you can use `tmux` or also use some other IDE feature to open multiple sessions in a single docker image.
I am okay with keeping these instructions if why we're doing it is mentioned.

Reply:
resolved